src/analysis/normed_space/basic.lean

```diff
@@ -1235,4 +1217,16 @@ summable_of_norm_bounded _ hf (assume i, le_refl _)
 lemma summable_of_summable_nnnorm {f : ι → α} (hf : summable (λa, nnnorm (f a))) : summable f :=
 summable_of_nnnorm_bounded _ hf (assume i, le_refl _)
 
+lemma real.summable_abs_iff {f : ι → ℝ} : summable (λ x, abs (f x)) ↔ summable f :=
```

Comment:
Is `normed_space.basic` the right file for this lemma?  It doesn't seem to use anything about normed spaces.  Maybe it could go in `data.real.*`?

Reply:
Thanks for noticing! Actually, I should move it to `topology/algebra/infinite_sum` and use proper typeclasses. My first proof was using normed space structure for one of the implications but this one doesn't need it.